static/js/shared/data_fetcher.ts

```diff
@@ -126,257 +84,58 @@ class StatsData {
       place = this.places[0];
     }
     const result: DataGroup[] = [];
-    for (const statsVar of this.statsVars) {
+    for (const statVar of this.statVars) {
       const dataPoints: DataPoint[] = [];
-      if (!this.data[statsVar][place]) continue;
-      const timeSeries = this.data[statsVar][place];
-      if (Object.keys(timeSeries.data).length !== 0) {
+      if (!this.data[place].data[statVar]) {
```

Comment:
is it guaranteed that this.data[place] won't be null or undefined? I think this will cause an error if this.data[place] is null or undefined.

Reply:
Done